migration.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Migrate to Pulp 3
+sidebar: home_sidebar
+permalink: /migrate-to-pulp-3/
+toc: false
+---
+
+## Migrate from Pulp 2 to Pulp 3
+
+For many years, the Pulp team have worked to gather and implement feedback from user deployments of Pulp 2. The solution to much of the feedback lay outside of the constraints of the Pulp 2 architecture. This led to the development of Pulp 3, which is not only an update of Pulp 2, but a complete technical overhaul to provide the most effective repository management platform.
+
+At the moment, the Pulp team supports both Pulp 2 and Pulp 3. However, at some point in the future, support will end for Pulp 2 and focus exclusively on Pulp 3.  
+
+If you are a Pulp 2 user, start planning your migration to Pulp 3.
+
+## Pulp 2 to Pulp 3 Migration Plugin
+
+You can use the migration plugin to migrate your content from Pulp 2 to Pulp 3. The development of this plugin is a work in progress. You can help support the development of this plugin by providing feedback on the different scenarios that you require to complete your migration. Currently, the migration plugin supports the following scenarios:
+
+*  Pulp 2 ISO can be migrated into Pulp 3 File.
+*  Pulp 2 Docker can be migrated into Pulp 3 Container.
+*  RPM plugin migration is planned and currently in development.
```

Comment:
Maybe worth mentioning that RPM migration is almost feature complete, there is 0.2.0 beta 2 which is available for the most curious ones. Up to you.

Reply:
+1 something along the lines:
RPM plugin migration is close to feature completion and will be released soon

Reply:
I added a line and a link to the beta 2 release notes. Thank you!